src/app/services/usuario.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { tap} from 'rxjs/operators';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
import { tap} from "rxjs/operators";
```